src/ConnectedMode/UI/ProjectSelection/ProjectSelectionWindow.xaml

```diff
@@ -0,0 +1,130 @@
+            <TextBlock Grid.Column="0" Grid.ColumnSpan="3" Grid.Row="1" IsHitTestVisible="False"
+                       Text="{x:Static ui:UiResources.SearchProjectPlaceholder}" VerticalAlignment="Center"
+                       HorizontalAlignment="Left" Margin="5,0,0,0" Foreground="DarkGray">
+                <TextBlock.Style>
+                    <Style TargetType="{x:Type TextBlock}">
+                        <Setter Property="Visibility" Value="Collapsed" />
+                        <Style.Triggers>
+                            <DataTrigger Binding="{Binding Text, ElementName=ProjectSearchTextBox}" Value="">
+                                <Setter Property="Visibility" Value="Visible" />
+                            </DataTrigger>
+                        </Style.Triggers>
+                    </Style>
+                </TextBlock.Style>
+            </TextBlock>
+
+            <ListBox Grid.Column="0" Grid.ColumnSpan="3" Grid.Row="2"
+                     Margin="0, 10, 0, 10"
+                     IsEnabled="{Binding !BindClicked}"
+                     ItemsSource="{Binding ProjectResults}"
+                     SelectedItem="{Binding SelectedProject}">
+                <ListBox.ItemTemplate>
+                    <DataTemplate>
+                        <StackPanel Orientation="Horizontal">
+                            <TextBlock Text="{Binding Name}" FontWeight="Bold" Margin="0,0,5,0" />
+                            <TextBlock>
+                                <TextBlock.Text>
+                                    <MultiBinding StringFormat="({0})">
+                                        <Binding Path="Key" />
+                                    </MultiBinding>
+                                </TextBlock.Text>
+                            </TextBlock>
+                        </StackPanel>
+                    </DataTemplate>
+                </ListBox.ItemTemplate>
+            </ListBox>
+
+            <Button Grid.Column="1" Grid.Row="3" Content="{x:Static ui:UiResources.BindButton}"
+                    HorizontalAlignment="Right"
+                    VerticalAlignment="Bottom"
+                    Margin="5, 0"
+                    IsEnabled="{Binding IsBindButtonEnabled}"
+                    Click="BindButton_OnClick" />
+            <Button Grid.Column="2" Grid.Row="3" Content="{x:Static ui:UiResources.CancelBtn}"
+                    HorizontalAlignment="Right"
+                    VerticalAlignment="Bottom"
+                    Click="CancelButton_OnClick" />
+        </Grid>
+        <DockPanel Grid.Row="1" LastChildFill="True">
+            <StatusBar DockPanel.Dock="Bottom">
+                <StatusBarItem>
+                    <TextBlock Text="{Binding BindingStatusText}"></TextBlock>
+                </StatusBarItem>
+                <StatusBarItem HorizontalContentAlignment="Right">
+                    <ProgressBar Height="15" Width="100" IsIndeterminate="{Binding BindClicked}" />
```

Comment:
The progresBar should always be indeterminate. So IsIndeterminate=true. Instead it should be hidden by default